src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,140 @@
 package com.theironyard.novauc;
 
+import java.util.HashMap;
+import java.util.Scanner;
+
 public class Main {
 
-    public static void main(String[] args) {
-	// write your code here
+    public static Scanner scanner = new Scanner(System.in);
+    public static Scanner scannerInt = new Scanner(System.in);
+    public static HashMap<String, Double> accountList = new HashMap<>();
+    public static String userName;
+
+
+    public static void main(String[] args) throws Exception {
+
+        //Scanner scanner = new Scanner(System.in);
+
+        //Put new user into the hashmap
+        accountList.put("steve", 30.26d);
+        accountList.put("jerry", -50.00d);
+
+
+        while (true) {
+            System.out.println("Welcome to the Uncommon Coders Atm. Please enter your User name:");
+            //fetch name from input
+            userName = scanner.nextLine().toLowerCase();
+            //See if user name is a key in our HashMap
+
+            if (!accountList.containsKey(userName)) {
+                System.out.println("You do not have an account with us. Press Y to create an account or press N to exit.");
+
+                String YorN = scanner.nextLine();
+
+                if (YorN.equalsIgnoreCase("Y")) {
+
+                    accountList.put(userName.toLowerCase(), 0.00);
+                    System.out.println("Welcome to the Uncommon Coders ATM " + userName + ".");
+                    System.out.println(userName + " your current balance is " + accountList.get(userName));
+                    // accountList.put(userName, 00.00);
+                } else if (!YorN.equalsIgnoreCase("Y")) {
+                    System.out.println("Goodbye");
+                    System.exit(0);
+                } else { //found an account
+                    //create variable to store bal
+                    double balance;
+                    //set balance variable to the value from HashMap
+                    balance = accountList.get(userName);
+                    System.out.println(" Your account balance is: " + balance);
+
```

Comment:
Good job Eric.  Take a look at else statement starting on  line 43.  I think it is not needed because if the user answers y the if statement runs otherwise an answer of not y the else if statement runs.  So there is really no way to access that code.